daphne_worker/src/lib.rs

```diff
@@ -182,14 +183,26 @@ pub struct DaphneWorkerReportSelector {
 }
 
 /// HTTP request handler for Daphne-Worker.
-#[derive(Default)]
-pub struct DaphneWorkerRouter {
+pub struct DaphneWorkerRouter<'a> {
```

Comment:
For consistency with lifetime used where this is called.
```suggestion
pub struct DaphneWorkerRouter<'srv> {
```